### PR TITLE
[WIP] public profile

### PR DESCRIPTION
**Pull request by Adhikram**
## Name of feature/Bug fixed/refactor

[Notion Task Link](https://www.notion.so/Public-Profile-of-a-User-2fd8f3e1998546cf88c86ccb040b18ef)

#### Branches for testing

Frontend: `feat/new-feature`

### CODE REVIEW

File: devsnest/Gemfile.lock [hunk 1]
```diff
@@ -179,6 +179,10 @@ GEM
       bundler (>= 1.3.0)
       railties (= 6.0.3.4)
       sprockets-rails (>= 2.0.0)
+    rails-controller-testing (1.0.5)
```
Comment (vedansh) [inline devsnest/Gemfile.lock:182]: why is this required

Comment (Adhikram) [reply devsnest/Gemfile.lock]: fixed

**Review by vedansh — APPROVED** [review]
